velox/common/base/SkewedPartitionBalancer.h

```diff
@@ -157,10 +174,29 @@ class SkewedPartitionRebalancer {
   // The estimated task processed bytes since the last rebalance.
   std::vector<uint64_t> estimatedTaskBytesSinceLastRebalance_;
 
-  // The assigned task id list for each partition.
-  std::vector<std::vector<uint32_t>> partitionAssignments_;
+  // The assigned task id assignment for a partition.
+  class PartitionAssignment {
```

Comment:
s/PartitionAssignment/PartitionAssigner/

Reply:
Do we need a class for this?

Reply:
This is to record the assignment state.